source/includes/_bulk-payments.md

```diff
@@ -23,6 +23,8 @@ endpoint can be used to see the current status of the batch payment instruction.
 
 The status of the money transfer is further broken down at a per investor level. 
 
+On batch state update a webhook will be fired that will contain the batch details with a summary of all payments. 
```

Comment:
missing comma -
'On a batch status update, a webhook..'

Btw - do you know if the 'GET' batch endpoint will be updated to point to this summary dto instead? kinda makes sense if it does - if yes, lines 21-22 need updating.